tests/attacks/inference/attribute_inference/test_black_box.py

```diff
@@ -229,5 +229,5 @@ def test_errors(art_warning, tabular_dl_estimator_for_attack, get_iris_dataset):
         art_warning(e)
 
 
-def test_classifier_type_check_fail():
-    backend_test_classifier_type_check_fail(AttributeInferenceBlackBox, (BaseEstimator, ClassifierMixin))
+# def test_classifier_type_check_fail():
+#     backend_test_classifier_type_check_fail(AttributeInferenceBlackBox, (BaseEstimator, ClassifierMixin))
```

Comment:
Is this change intended?

Reply:
No, sorry about that. I committed the wrong file. Undoing now.